codesystems/CodeSystem-England-FGMRemovalReason.xml

```diff
@@ -23,7 +23,8 @@
   </contact>
   <description value="A CodeSystem used for the removal of FGM Flag" />
   <purpose value="A codesystem used for the removal of FGM Flag" />
-  <copyright value="Copyright © 2023+ NHS England Licensed under the Apache License, Version 2.0 (the \&quot;License\&quot;); you may not use this file except in compliance with the License. You may obtain a copy of the License at  http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an \&quot;AS IS\&quot; BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License. HL7® FHIR® standard Copyright © 2011+ HL7 The HL7® FHIR® standard is used under the FHIR license. You may obtain a copy of the FHIR license at  https://www.hl7.org/fhir/license.html." />
+  <copyright
```

Comment:
Not sure what you tried to change here but should be just over one line (new line has been introduced)